src/app/kvm/components/LXDHostVMs/LXDHostVMs.tsx

```diff
@@ -54,7 +55,7 @@ const LXDHostVMs = ({
 
   if (pod) {
     return (
-      <>
+      <div {...wrapperProps}>
```

Comment:
There are a number of components that had little updates to support the React Testing Library tests.